http_service/app.py

```diff
@@ -43,16 +97,50 @@ def model_prediction(model, bug_id):
     else:
         LOGGER.info("Request with API TOKEN %r", auth)
 
-    bugs = bugzilla._download([bug_id])
-    model = get_model(model)
-    probs = model.classify(list(bugs.values()), True)
-    indexes = probs.argmax(axis=-1)
-    suggestions = model.clf._le.inverse_transform(indexes)
+    status_code = 200
+    data = get_bug_classification(model_name, bug_id)
+
+    if not data:
+        schedule_bug_classification(model_name, [bug_id])
+        status_code = 202
+        data = {"ready": False}
+
+    return jsonify(**data), status_code
+
+
+@application.route("/<model_name>/predict/batch", methods=["POST"])
+def batch_prediction(model_name):
+    headers = request.headers
+
+    auth = headers.get(API_TOKEN)
+
+    if not auth:
+        return jsonify({"message": "Error, missing X-API-KEY"}), 401
+    else:
+        LOGGER.info("Request with API TOKEN %r", auth)
+
+    # TODO Check is JSON is valid and validate against a request schema
+    batch_body = json.loads(request.data)
+
+    bugs = batch_body["bugs"]
+
+    status_code = 200
+    data = {}
+    missing_bugs = []
+
+    for bug in bugs:
+        data[str(bug)] = get_bug_classification(model_name, bug)
+        if not data[str(bug)]:
```

Comment:
What happens if we have a security bug? Will we always return 202?

Reply:
No because we add an error placeholder for missing bugs here: https://github.com/mozilla/bugbug/pull/321/files/225b0a03969aeb5c63c064dec0215e18a632b59f#diff-7908c821177e4af7e08af4a9ca055d60R88

It can be security bugs or non-existing bug ids